### ISSUE TITLE
Cider stops working when clojars is offline

### ISSUE THREAD

**Issue by edvorg** [clojure-emacs/cider#1538]
Hello.
I think that it might be considered as a bug.
Here are some details from logs. Trying to jack-in in my app.

```
Feb 03, 2016 12:13:32 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.NoRouteToHostException) caught when processing request to {s}->https://clojars.org:443: No route to host
Feb 03, 2016 12:13:32 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {s}->https://clojars.org:443
Feb 03, 2016 12:14:48 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.NoRouteToHostException) caught when processing request to {s}->https://clojars.org:443: No route to host
Feb 03, 2016 12:14:48 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {s}->https://clojars.org:443
Feb 03, 2016 12:16:03 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.NoRouteToHostException) caught when processing request to {s}->https://clojars.org:443: No route to host
Feb 03, 2016 12:16:03 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {s}->https://clojars.org:443
Feb 03, 2016 12:18:34 AM org.apache.http.impl.execchain.RetryExec execute
INFO: I/O exception (java.net.NoRouteToHostException) caught when processing request to {s}->https://clojars.org:443: No route to host
Feb 03, 2016 12:18:34 AM org.apache.http.impl.execchain.RetryExec execute
INFO: Retrying request to {s}->https://clojars.org:443

```

**Comment by expez** [comment]
That's from clj-refactor.  This has been fixed in refactor-nrepl 2.0.0-SNAPSHOT.

**Comment by edvorg** [comment]
Thank you!
